_data/engine-cli-edge/docker_service_create.yaml

```diff
@@ -383,813 +394,354 @@ options:
-  `--credential-spec` must be in the format `file://<filename>` or
-  `registry://<value-name>`.
-
-  When using the `file://<filename>` format, the referenced file must be
-  present in the `CredentialSpecs` subdirectory in the docker data directory,
-  which defaults to `C:\ProgramData\Docker\` on Windows. For example,
-  specifying `file://spec.json` loads `C:\ProgramData\Docker\CredentialSpecs\spec.json`.
-
-  When using the `registry://<value-name>` format, the credential spec is
-  read from the Windows registry on the daemon's host. The specified
-  registry value must be located in:
-
-      HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Virtualization\Containers\CredentialSpecs
-
-
-  ### Create services using templates
-
-  You can use templates for some flags of `service create`, using the syntax
-  provided by the Go's [text/template](http://golang.org/pkg/text/template/) package.
-
-  The supported flags are the following :
-
-  - `--hostname`
-  - `--mount`
-  - `--env`
-
-  Valid placeholders for the Go template are listed below:
-
-
-  <table>
-    <tr>
-      <th>Placeholder</th>
-      <th>Description</th>
-    </tr>
-    <tr>
-      <td><tt>.Service.ID</tt></td>
-      <td>Service ID</td>
-    </tr>
-    <tr>
-      <td><tt>.Service.Name</tt></td>
-      <td>Service name</td>
-    </tr>
-    <tr>
-      <td><tt>.Service.Labels</tt></td>
-      <td>Service labels</td>
-    </tr>
-    <tr>
-      <td><tt>.Node.ID</tt></td>
-      <td>Node ID</td>
-    </tr>
-    <tr>
-      <td><tt>.Node.Hostname</tt></td>
-      <td>Node Hostname</td>
-    </tr>
-    <tr>
-      <td><tt>.Task.ID</tt></td>
-      <td>Task ID</td>
-    </tr>
-    <tr>
-      <td><tt>.Task.Name</tt></td>
-      <td>Task name</td>
-    </tr>
-    <tr>
-      <td><tt>.Task.Slot</tt></td>
-      <td>Task slot</td>
-    </tr>
-  </table>
-
-
-  #### Template example
-
-  In this example, we are going to set the template of the created containers based on the
-  service's name, the node's ID and hostname where it sits.
-
-  ```bash
-  $ docker service create --name hosttempl \
-                          --hostname="{{.Node.Hostname}}-{{.Node.ID}}-{{.Service.Name}}"\
-                           busybox top
-
-  va8ew30grofhjoychbr6iot8c
-
-  $ docker service ps va8ew30grofhjoychbr6iot8c
-
-  ID            NAME         IMAGE                                                                                   NODE          DESIRED STATE  CURRENT STATE               ERROR  PORTS
-  wo41w8hg8qan  hosttempl.1  busybox:latest@sha256:29f5d56d12684887bdfa50dcd29fc31eea4aaf4ad3bec43daf19026a7ce69912  2e7a8a9c4da2  Running        Running about a minute ago
-
-  $ docker inspect --format="{{.Config.Hostname}}" 2e7a8a9c4da2-wo41w8hg8qanxwjwsg4kxpprj-hosttempl
-
-  x3ti0erg11rjpg64m75kej2mz-hosttempl
-  ```
+examples: "### Create a service\n\n```bash\n$ docker service create --name redis redis:3.0.6\n\ndmu1ept4cxcfe8k8lhtux3ro3\n\n$
```

Comment:
@thaJeztah Do you know why this happened? It doesn't seem to affect output but it sure is ugly. Saw it in a few other places too.

Reply:
Ah, yes, I think I discovered why that happens; see https://github.com/docker/cli/pull/564#issue-260898433

If a text-block contains trailing whitespace, YAML uses a different approach to encode it. Trimming the whitespace before generating the YAML forces it to use the `|-` "block chomp" (yes, I learned that when working on that PR)

I see there was a comment from Daniel on that PR that I should check 😅